.github/workflows/cancel.yaml

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Cancel
+
+on:
+  workflow_run:
+    workflows:
+      - CI AND IT
+      - E2E
+      - PluginsTest
+    types:
+      - requested
+
+jobs:
+  cancel-outdated-builds:
```

Comment:
Could you share what cancel tests could do now? And add those add comments here?

Reply:
Compared to https://github.com/apache/skywalking/issues/5556#issuecomment-698836978 , now the improvement is

![image](https://user-images.githubusercontent.com/15965696/97805721-d3d9c780-1c92-11eb-90ef-263ce457b23d.png)

this is helpful whenever the `cancel-outdated-builds` job get a slot to run, it cancels **ALL** outdated builds, no matter which PR it is running for. For example, if PR 1 has 2 jobs in queue, where 1 jobs are outdated, and PR 2 has 100 jobs in queue, where 99 jobs are outdated, when PR1 triggers a `cancel-outdated-builds` to run, it will cancel outdated jobs in both PR1 and PR2, leaving only 2 latest jobs to run for PR1 and PR2 respectively.

Reply:
OK, it will scan all outdates jobs in all PRs. Then it only requires one slot, then the blocking queue could be clear as much as possible.

Reply:
Could you add some comments about this?

```suggestion
  # Cancel outdated builds in the repo, including all action tasks of all commits.
  cancel-outdated-builds:
```